lib/activerecord-tenant-level-security/schema_statements.rb

```diff
@@ -1,22 +1,22 @@
 module TenantLevelSecurity
   module SchemaStatements
-    def create_policy(table_name)
+    def create_policy(table_name, column_name: 'tenant_id')
```

Comment:
It's a good idea to mention in the README what your partition key default is and that it can be changed by this option.

Reply:
Thank you, I added it in this commit https://github.com/kufu/activerecord-tenant-level-security/pull/22/commits/3a26d9b6e52c4ad9ea92b773a1ae86faf83485ba